po/es.po

```diff
@@ -0,0 +1,530 @@
+# Spanish translations for Game Conqueror.
+# Copyright (C) 2009-2014 Wang Lu <coolwanglu(a)gmail.com>
+# This file is distributed under the same license as Game Conqueror.
+# Salvador Pardiñas <darkfm(at)vera.com.uy>, 2014.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: scanmem 0.14\n"
```

Comment:
The metadata here are a bit messed up, current version is v0.16 and the revision date of this translation sure isn't in 2014.